src/frontends/pytorch/src/transforms/prim_tuple_construct_replacer.cpp

```diff
@@ -29,7 +29,19 @@ bool DecomposeTupleResults::run_on_model(const std::shared_ptr<Model>& model) {
         }
         auto inputs = input_node->inputs();
         for (auto input : inputs) {
-            model->add_results({std::make_shared<opset10::Result>(input.get_source_output())});
+            auto out = input.get_source_output();
+            auto list_construct = cast_fw_node(out.get_node_shared_ptr(), "prim::ListConstruct");
```

Comment:
Question: if we do not do this, after we decompose `TupleConstruct` we will have `ListConstruct` on the output, can we decompose `ListConstruct` independently of `TupleConstruct`? If we had model that have `ListConstruct` instead of `TupleConstruct` on the output that will allow it to convert too